client/styles/LandingHeader.module.css

```diff
@@ -1,18 +1,19 @@
 .landingHeader {
-    height: 450px;
     display: flex;
     background-image: url("public/landing-back-img.png");
     background-position: center;
     background-repeat: no-repeat;
     background-size: cover;
     justify-content: center;
     align-items: center;
+    width: 100vw;
```

Comment:
`100vw` shouldn't be necessary if the parent elements fill the viewport

Reply:
This is the parent element on the page?

Reply:
It has a wrapper div, and there are other parents in the document. It's not too important for now.